autofit/non_linear/abstract_search.py

```diff
@@ -289,25 +287,12 @@ def optimise(
             result.projected_model.priors
         )
 
-        variable_message_count = model_approx.variable_message_count
-        min_value = min(variable_message_count.values())
-
-        if self.dynamic_delta:
-            delta = MeanField({
-                variable: self.delta * (min_value / message_count)
-                for variable, message_count in variable_message_count.items()
-            })
-        else:
-            delta = self.delta
-
-        projection, status = factor_approx.project(

```

Comment:
This projection step does not occur anymore. @matthewghgriffiths is this correct?